README.md

```diff
@@ -96,145 +97,61 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **colored header**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+> Not avaiable yet
```

Comment:
***[OPTIONAL]*** `Live demo link` means the live version of your website that can be accessed from anywhere in the world. 
Kindly consider adding the link as it is professional, and the viewer can easily find your website's online version from Readme. Here is a [FreecodeCamp Link](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) for reference.